 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 272-898-683
127.0.0.1 - - [20/May/2017 13:22:04] "GET / HTTP/1.1" 200 -
 * Detected change in '/home/alisaleemh/Development/menu-site/project.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 272-898-683
127.0.0.1 - - [20/May/2017 13:22:18] "GET / HTTP/1.1" 200 -
 * Detected change in '/home/alisaleemh/Development/menu-site/project.py', reloading
 * Restarting with stat
  File "project.py", line 31
    if __name__ == '__main__':
     ^
SyntaxError: invalid syntax
  File "project.py", line 30
    def
      ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "project.py", line 17, in <module>
    @app.route('/restaurants/<int: restaurant_id>/')
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1080, in decorator
    self.add_url_rule(rule, endpoint, f, **options)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 64, in wrapper_func
    return f(self, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1046, in add_url_rule
    self.url_map.add(rule)
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/routing.py", line 1207, in add
    rule.bind(self)
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/routing.py", line 687, in bind
    self.compile()
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/routing.py", line 737, in compile
    _build_regex(self.is_leaf and self.rule or self.rule.rstrip('/'))
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/routing.py", line 713, in _build_regex
    for converter, arguments, variable in parse_rule(rule):
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/routing.py", line 210, in parse_rule
    raise ValueError('malformed url rule: %r' % rule)
ValueError: malformed url rule: '/restaurants/<int: restaurant_id>'
